src/core/AudioEngine.cpp

```diff
@@ -608,22 +613,22 @@ void AudioEngine::changeQuality(const struct qualitySettings & qs)
 
 
 
-void AudioEngine::doSetAudioDevice( AudioDevice * _dev )
+void AudioEngine::doSetAudioDevice(AudioDevice * _dev)
 {
 	// TODO: Use shared_ptr here in the future.
 	// Currently, this is safe, because this is only called by
 	// ProjectRenderer, and after ProjectRenderer calls this function,
 	// it does not access the old device anymore.
-	if( m_audioDev != m_oldAudioDev ) {delete m_audioDev;}
+	if (m_audioDev != m_oldAudioDev) { delete m_audioDev; }
 
-	if( _dev )
+	if (_dev)
 	{
 		m_audioDev = _dev;
 	}
 	else
 	{
-		printf( "param _dev == NULL in AudioEngine::setAudioDevice(...). "
-					"Trying any working audio-device\n" );
+		printf("param _dev == NULL in AudioEngine::setAudioDevice(...). "
+					"Trying any working audio-device\n");
```

Comment:
I would remove these calls to `printf`. The point in code where we should do this is mainly when an exception is thrown, but our error handling still needs some work.
```suggestion
```